src/System.IO.Compression/src/System/IO/Compression/ZLibException.Serialization.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using System.IO.Compression;
+
+namespace System.IO.Compression
+{
+    [Serializable]
+    internal partial class ZLibException : IOException, ISerializable
```

Comment:
>ISerializable [](start = 56, length = 13)

I suppose we can't do similar as the Serializable attribute here and create a emtpy ISerializable interface for non-netstandard1.7? Probably is not worth it though...